commands/update_stemcell.go

```diff
@@ -3,21 +3,21 @@ package commands
 import (
 	"fmt"
 	"log"
+	"strings"
 
 	"github.com/pivotal-cf/jhanda"
-	"github.com/pivotal-cf/kiln/builder"
 	"github.com/pivotal-cf/kiln/fetcher"
-	"github.com/pivotal-cf/kiln/helper"
 	"github.com/pivotal-cf/kiln/release"
+	"github.com/Masterminds/semver"
 	"gopkg.in/src-d/go-billy.v4/osfs"
```

Comment:
(feel free to ignore this comment)

I usually sort my import statements into these ordered blocks (based on [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) but extended a little bit)
- standard library
- dot imports (https://github.com/golang/go/wiki/CodeReviewComments#import-dot)
- test helper/matcher packages
- external libraries
- packages in the current module

Reply:
Yhea I liked this. Usually I run tools for other languages that does this for me. Probably I need to setup something for golang.

For now I manually updated the imports.